model/parts/types/graph.go

```diff
@@ -34,10 +33,6 @@ type EdgeAttrs struct {
 	Threshold int
 }
 
-//func (g *Graph) FindResponsibleNodes(chunkId int) [4]int {
-//	return g.RespNodes[chunkId]
-//}
-
 func BinarySearchClosest(arr []NodeId, target int, n int) ([]NodeId, NodeId) {
```

Comment:
This function `BinarySearchClosest` should probably also be removed.
I believe it is not used for anything but finding responsible nodes.

Reply:
Can you please check now.

Reply:
Nice, please merge now